luigi_pipeline/seqr_loading_optimized.py

```diff
@@ -42,37 +41,49 @@ def run(self):
         if self.subset_path:
             mt = self.subset_samples_and_variants(mt, self.subset_path)
 
-        mt = SeqrGenotypesSchema(mt).annotate_all(overwrite=True).select_annotated_mt()
+        mt = self.GenotypesSchema(mt).annotate_all(overwrite=True).select_annotated_mt()
 
         mt.describe()
         mt.write(self.output().path, stage_locally=True, overwrite=True)
 
 
-class SeqrMTToESOptimizedTask(HailElasticSearchTask):
+class SeqrVCFToGenotypesMTTask(BaseVCFToGenotypesMTTask):
+    VariantTask = SeqrVCFToVariantMTTask
+    GenotypesSchema = SeqrGenotypesSchema
+
+
+class BaseMTToESOptimizedTask(HailElasticSearchTask):
 
     def __init__(self, *args, **kwargs):
         # TODO: instead of hardcoded index, generate from project_guid, etc.
         super().__init__(*args, **kwargs)
 
     def requires(self):
-        return [SeqrVCFToVariantMTTask(), SeqrVCFToGenotypesMTTask()]
+        return [self.VariantTask(), self.GenotypesTask()]
 
     def run(self):
         variants_mt = hl.read_matrix_table(self.input()[0].path)
         genotypes_mt = hl.read_matrix_table(self.input()[1].path)
+        genotypes_mt = genotypes_mt.drop(*[k for k in genotypes_mt.globals.keys()])
         row_ht = genotypes_mt.rows().join(variants_mt.rows())
 
-        row_ht = SeqrVariantsAndGenotypesSchema.elasticsearch_row(row_ht)
+        row_ht = self.VariantsAndGenotypesSchema.elasticsearch_row(row_ht)
         es_shards = self._mt_num_shards(genotypes_mt)
 
         # Initialize an empty SeqrVariantsAndGenotypesSchema to access class properties
-        disabled_fields = SeqrVariantsAndGenotypesSchema(None, ref_data=defaultdict(dict), clinvar_data=None).get_disable_index_field()
-        
+        disabled_fields = self.VariantsAndGenotypesSchema(None, ref_data=defaultdict(dict), clinvar_data=None).get_disable_index_field()
```

Comment:
This is a change to the new code for adding the `disable_index` option.

Reply:
I don't see how this is a change? This seems like just the change you make everywhere in this class to use the correct base class

Reply:
`SeqrVariantsAndGenotypesSchema` is changed to `self.VariantsAndGenotypesSchema`. The git merged the code and reported no conflicts. I made the change manually.